### PR TITLE
[WiP] add possibility to configure lacp bonding at install

### PR DESCRIPTION
**Pull request by benjamreis**

### CODE REVIEW

**Comment by benjamreis** [comment]
https://github.com/xcp-ng/xcp/issues/350

File: netinterface.py [hunk 1]
```diff
@@ -69,6 +69,13 @@ def __init__(self, mode, hwaddr, ipaddr=None, netmask=None, gateway=None,
         self.ipv6addr = None
         self.ipv6_gateway = None
 
+        if bond_mode is not None:
+            # Not `balance-slb` because it's openvswitch specific
+            assert bond_mode in ["lacp", "active-backup"]
```
Comment (stormi) [inline netinterface.py:74]: What if the users provided a different value? How will the error be displayed to them?

**Review by stormi — COMMENTED** [review]
Given the amount of changes, it becomes really important to find a way to contribute it upstream, to avoid merge conflicts each time they change code around this.

**Comment by benjamreis** [comment]
Yes upstream our changes would be nice.
Especially since nothing is us-specific in terms of features. :)

File: netinterface.py (hunk 1 above)
Comment (benjamreis) [reply netinterface.py:74]: idk, i've reproduced how such errors are handled in the code.
The UI won't allow to give other values. The answerfiles think will be handled in `answerfile.py` so this is not where to treat user error display.

Comment (stormi) [reply netinterface.py:74]: Ok, so this means you'll sanitize answerfile input so that those asserts are never reached?

Comment (benjamreis) [reply netinterface.py:74]: Yes, once i'm tackling this part in the parsong of the answerfile.

**Comment by benjamreis** [comment]
Obsoleted by https://github.com/xcp-ng/host-installer/pull/9

**Pull request closed without merging**